docs/storage.md

```diff
@@ -61,17 +61,20 @@ A Prometheus server's data directory looks something like this:
 Note that a limitation of local storage is that it is not clustered or
 replicated. Thus, it is not arbitrarily scalable or durable in the face of
 drive or node outages and should be managed like any other single node
-database. 
+database.
 
 [Snapshots](querying/api.md#snapshot) are recommended for backups. Backups 
 made without snapshots run the risk of losing data that was recorded since 
-the last WAL sync, which typically happens every two hours. With proper
-architecture, it is possible to retain years of data in local storage.
-
-Alternatively, external storage may be used via the
-[remote read/write APIs](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage).
-Careful evaluation is required for these systems as they vary greatly in durability,
-performance, and efficiency.
+the last WAL sync, which typically happens every two hours, covering
```

Comment:
Not your change, but "sync" makes me think of Unix `sync(1)`, which Prometheus will do every time it creates a new WAL segment, generally rather less than 2 hours.

The operation that happens every 2 hours is a "head compaction", followed by a "checkpoint" and "WAL truncation".
I think "WAL checkpoint" would convey the point without being misleading.

Reply:
changed, thanks!